### PR TITLE
[Project Submission] MyWorld Project

### PR DESCRIPTION
**Pull request by NguyenIvan**
MyWorld is a common place for autism artists to express their world through arts and music. They can sell and exchange their work via the smart contracts. A small fee from those buying and selling is contributed to autism awareness activies; those activities in turn are proposed and voted by mean of token distribution in the community. All those beautiful things happend thanks to the greatness of Flow blockchain network, NFT.storage API and IPFS storage space.

### CODE REVIEW

**Comment by NguyenIvan** [comment]
@jimpick: Can you please review my project? Thank you.

File: projects/application/MyWorld.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+## Project Name MyWorld<!-- Add your project name here with format "Project Name"-->
+
+## Category 
+<!--developer tooling, application, wallet, infrastructure, etc-->
+Application
```
Comment (longfeiWan9) [inline projects/application/MyWorld.md:5]:
```suggestion

Application
```

**Comment by longfeiWan9** [comment]
@NguyenIvan Awesome job.

I reviewed this PR and added few suggestions to fix the format of your MD file. Please take a look at them.

BTW, this is a great project to explore the possibility of NFT and IPFS, also support autism artists. We would love to support more projects like this. If you and the team is looking for funding to move forward with the project, I highly recommend you take a look at Fielcoin Next Step Microgrant, [link is here](https://github.com/filecoin-project/devgrants/blob/master/microgrants/microgrants.md).

**Comment by NguyenIvan** [comment]
@longfeiWan9  thank you for your comment. I am still trying to find you code comments, can you help me to find them please?
To succeed with the project I have much more to learn from filecoin team and flow team. It is a great honor for us if we are in one of your programs, and Next Step Microgrant is an amazing opportunity. We will try our best.
I guess next step, if everything is OK, is to merge our PR? If so, please help us move forward.

**Comment by longfeiWan9** [comment]
@NguyenIvan Can you see it now ?

**Comment by NguyenIvan** [comment]
I changed accordingly.

**Review by longfeiWan9 — APPROVED** [review]

**Comment by NguyenIvan** [comment]
Nice, thank you @longfeiWan9

**Pull request merged**